pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -1750,3 +1756,22 @@ type AuthorizationValidationError struct {
 func (e *AuthorizationValidationError) Error() string {
 	return e.err
 }
+
+type ScrapeClass struct {
+	// Name of the scrape class, e.g. "istio-mtls".
```

Comment:
```suggestion
	// Name of the scrape class.
```

I know the istio-mtls is a common use case, but I don't see a reason for keeping it as example